src/components/TodoList/TodoList.js

```diff
@@ -1,41 +1,69 @@
 import React from 'react';
 import './TodoList.css';
 
-function handleClick() {
-  console.log(this)
-}
-
 class TodoList extends React.Component {
 
-  // state = {
-  //   list: [
-  //     { id: 1, name: 'Fumer des cannes', active: true },
-  //     { id: 2, name: 'Coder cette shit', active: true },
-  //     { id: 3, name: 'Nager avec des baleines', active: true },
-  //     { id: 4, name: 'Rouler des cannes pour demain', active: true }
-  //   ],
-  //   newTaskName: "",
-  // // }
-  //
-  // handleClick = () => {
-  //   console.log(this)
-  // }
+  state = {
+    list: [ ],
```

Comment:
To create an empty list, you simply do `list: []` instead of `list: [ ]` (with the whitespace between `[` and `]`). You will always see `[]` in the code